initializr-service/src/main/resources/application.yml

```diff
@@ -22,6 +22,14 @@ spring:
 initializr:
   env:
     boms:
+      spring-cloud-gcp:
```

Comment:
Please order the BOM according to its id (checking other entries for reference).